alvr/client_openxr/src/interaction.rs

```diff
@@ -371,9 +375,12 @@ impl InteractionContext {
 
     pub fn select_sources(&mut self, config: &InteractionSourcesConfig) {
         // First of all, disable/delete all sources. This ensures there are no conflicts
-        extra_extensions::pause_simultaneous_hands_and_controllers_tracking_meta(&self.xr_session)
-            .ok();
-        self.multimodal_hands_enabled = false;
+        if let Some((handle, enabled)) = &mut self.multimodal_hands {
+            if *enabled {
+                handle.pause().ok();
+                *enabled = false;
+            }
+        }
```

Comment:
If we just keep the multimodal_enables flag as a bool the this could become
```rust
self.multimodal_hands.map(|m| m.pause().ok());
enabled = false
```
Where we just do it unconditionally if we have multimodal support.

Reply:
Advantage of always pausing it, even if it should already be paused is that we're not gonna run into any scenarios where it's still running for some reason and then we try to enable body tracking or something.
Since I'm pretty sure the issue wasn't pausing when it wasn't enabled, the issue was doing the xr call on devices that don't support it.

Reply:
yup, we discussed this before